src/hooks/index.js

```diff
@@ -674,29 +810,46 @@ function validatedAttachments(attachments) {
 // mail.html = generatedHtml || "";
 exports.sendEmail = (context, maildata) => {
 	const files = maildata.attachments || [];
-	if (files) { validatedAttachments(files); }
+	if (files) {
+		validatedAttachments(files);
+	}
 
 	const userService = context.app.service('/users');
 	const mailService = context.app.service('/mails');
 
-	const roles = (typeof maildata.roles === 'string' ? [maildata.roles] : maildata.roles) || [];
-	const emails = (typeof maildata.emails === 'string' ? [maildata.emails] : maildata.emails) || [];
-	const userIds = (typeof maildata.userIds === 'string' ? [maildata.userIds] : maildata.userIds) || [];
+	const roles =
+		(typeof maildata.roles === 'string'
+			? [maildata.roles]
+			: maildata.roles) || [];
+	const emails =
+		(typeof maildata.emails === 'string'
+			? [maildata.emails]
+			: maildata.emails) || [];
+	const userIds =
+		(typeof maildata.userIds === 'string'
+			? [maildata.userIds]
+			: maildata.userIds) || [];
 	const receipients = [];
 
 	// create email attachments
 	const attachments = [];
 	files.forEach((element) => {
 		try {
-			attachments.push({ filename: element.name, content: Buffer.from(element.data) });
+			attachments.push({
+				filename: element.name,
+				content: Buffer.from(element.data),
+			});
 		} catch (error) {
 			throw new Error('Unexpected Error while creating Attachment.');
 		}
 	});
 
 	// email validation conform with <input type="email"> (see https://emailregex.com)
 	const re = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
-	const replyEmail = ((maildata.replyEmail) && re.test(maildata.replyEmail)) ? maildata.replyEmail : null;
+	const replyEmail =
```

Comment:
Codacy found an issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)